src/core.js

```diff
@@ -28,16 +28,22 @@ const getContext = (context) => {
 
 /**
  * @typedef {object} Watch
```

Comment:
Was `src/build.js` left empty on purpose, or is it dead code?

Reply:
Good catch, I initially thought about adding a more sophisticated logic (like we have in `watch`), but decided to keep it simple for now and use the same logic as in our UI